docs/nodes/node-maintenance-and-security.md

```diff
@@ -0,0 +1,176 @@
+* Node private keyring: `/var/ton-work/db/keyring`
+* Node public keys: `/var/ton-work/keys`
+* Mytonctrl configuration and wallets: `$HOME/.local/share/myton*` where $HOME is home directory of user you started installation of mytonctrl from. **OR** `/usr/local/bin/mytoncore` if you installed mytonctrl as root.
+
+This set is everything you need to perform recovery of your node from scratch.
+
+#### Snapshots
+Modern file systems such as ZFS offer snapshot functionality, most cloud providers also allow their customers to make snapshots of machines during which entire disk is preserved for future use.
+
+The problem with both methods is that you must stop node before performing a snapshot, failure to do so will most likely result in corrupt database with unexpected consequences. Many cloud providers also require you do power down the machine before performing a snapshot. 
+
+Such stops should not be performed often, if you snapshot your node once a week then in the worst case scenario after recovery you will have a node with week old database and it will take your node more time to catch up with the network then to perform a new installation using mytonctrl "install from dump" feature (-d flag added during invocation of `install.sh` script).
+
+### <a id="disaster-recovery"></a>Disaster recovery
+To perform recovery of your node on a new machine:
+
+#### Install mytonctrl / node 
+For fastest node initialization add `-d` switch to invocation of installation script.
+
+#### Switch to root user
+```sh
+sudo -s
+```
+#### Stop mytoncore and validator processes
+```sh
+systemctl stop validator
+systemctl stop mytoncore
+```
+#### Apply backed up node configuration files
+* Node configuration file: `/var/ton-work/db/config.json`
+* Node private keyring: `/var/ton-work/db/keyring`
+* Node public keys: `/var/ton-work/keys`
+
+#### <a id="set-node-ip"></a> Set node IP address
+If your new node has a different IP address then you must edit node configuration file `/var/ton-work/db/config.json` and set the leaf `.addrs[0].ip` to **decimal** representation of new IP address. You can use **[this](https://github.com/sonofmom/ton-tools/blob/master/node/ip2dec.py)** python script to convert your IP to decimal.
+
+#### Ensure proper database permissions
+```sh
+chown -R validator:validator /var/ton-work/db
+```
+#### Apply backed up mytonctrl configuration files
+Replace `$HOME/.local/share/myton*` where $HOME is home directory of user you started installation of mytonctrl from with backed up content, make sure that the user is owner of all files you copy.
+
+#### Start mytoncore and validator processes
+```sh
+systemctl start validator
+systemctl start mytoncore
+```
+## <a id="security"></a>Security
+### <a id="host-security"></a>Host level security
+Host level security is huge topic that lies outside of the scope of this document, we do however advise to never install mytonctrl under root user, do use service account to ensure priveledge separation.
```

Comment:
Host level security is huge topic that lies outside of the scope of this document, we do however advise to never install mytonctrl under root user, do use service account to ensure privilege separation.